src/plugins/legacy/vtkDataMesh/CMakeLists.txt

```diff
@@ -31,8 +31,9 @@ add_definitions(-D${TARGET_NAME_UP}_VERSION="${${TARGET_NAME}_VERSION}")
 find_package(dtk REQUIRED)
 include_directories(${dtk_INCLUDE_DIRS})
 
-find_package(VTK REQUIRED COMPONENTS vtkCommonCore vtkCommonDataModel vtkRenderingCore vtkRenderingVolumeOpenGL2 vtkInteractionStyle vtkInteractionWidgets vtkIOParallelXML vtkIOImage vtkIOGeometry  vtkIOExport vtkRenderingContext2D)
-include(${VTK_USE_FILE})
+find_package(VTK REQUIRED COMPONENTS CommonCore CommonDataModel RenderingCore RenderingVolumeOpenGL2 
+  InteractionStyle InteractionWidgets IOParallelXML IOImage IOGeometry IOExport RenderingContext2D)
+include_directories(${VTK_INCLUDE_DIRS})
```

Comment:
VTK no longer defines `VTK_INCLUDE_DIRS`, and in general most CMake packages do not use this mechanism any more. The include directories are added to the imported targets using the `INCLUDE_DIRECTORIES` property, so linking with them is sufficient. Also VTK provides `VTK_LIBRARIES` to simplify linking (no need to list all the library targets as this variable will hold all the ones defined as components in the `find_package` call).